### PR TITLE
WIP: Net standard

### PR DESCRIPTION
**Pull request by PolitovArtyom**
How about this?

For .Net standard 1.0 support we should remove DuplicateTypeMappingException. And it will break backward compatibility.,

### CODE REVIEW

File: src/Unity.RegistrationByConvention.csproj [hunk 1]
```diff
@@ -19,33 +19,28 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>package.snk</AssemblyOriginatorKeyFile>
     <DelaySign>false</DelaySign>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;netstandard2.0;net47;net46;net45</TargetFrameworks>
```
Comment (davidvedvick) [inline src/Unity.RegistrationByConvention.csproj:22]: Is there any reason you can't just target `netstandard2.0`?

**Comment by ENikS** [comment]
Why don't you conditionally provide this exception for Netstandard 1.0? Just cut and paste implementation and include it just for that platform.

**Comment by yfital** [comment]
Hey @danrien,

Do we have an ETA for the merge request? the package contains a lot of static functions which are very handy

I don't think netstnd1.x should be blocking here, support can be added later.
If there is any help / testing needed before merging, i'll be happy to help

**Comment by PolitovArtyom** [comment]
@ENikS 
As i know, netstandard 1.0 has no support for serializable attribute. So there are no safe way to use custom exceptions in nuget packages due to problems while working with cross-domain applications.

**Comment by ENikS** [comment]
Thank you for PR, I've deployed new package.
Feel free to send more of these...

**Pull request merged**